src/main/archetype/core/src/test/java/core/filters/LoggingFilterTest.java

```diff
@@ -15,36 +15,34 @@
  */
 package ${package}.core.filters;
 
+import java.io.IOException;
+import java.util.List;
+
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+
 import org.apache.sling.testing.mock.sling.servlet.MockRequestPathInfo;
 import org.apache.sling.testing.mock.sling.servlet.MockSlingHttpServletRequest;
 import org.apache.sling.testing.mock.sling.servlet.MockSlingHttpServletResponse;
-import org.junit.Rule;
-import org.junit.Test;
-import io.wcm.testing.mock.aem.junit.AemContext;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import io.wcm.testing.mock.aem.junit5.AemContext;
+import io.wcm.testing.mock.aem.junit5.AemContextExtension;
 import uk.org.lidalia.slf4jext.Level;
 import uk.org.lidalia.slf4jtest.LoggingEvent;
 import uk.org.lidalia.slf4jtest.TestLogger;
 import uk.org.lidalia.slf4jtest.TestLoggerFactory;
-import uk.org.lidalia.slf4jtest.TestLoggerFactoryResetRule;
-
-import javax.servlet.FilterChain;
-import javax.servlet.FilterConfig;
-import javax.servlet.ServletException;
-
-import java.io.IOException;
-import java.util.List;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.mockito.Mockito.mock;
-import static org.junit.Assert.*;
 
+@ExtendWith(AemContextExtension.class)
 public class LoggingFilterTest {
 
-    @Rule
     public final AemContext context = new AemContext();
 
-    @Rule
-    public final TestLoggerFactoryResetRule testLoggerFactoryResetRule = new TestLoggerFactoryResetRule();
-
```

Comment:
I notice you removed this but didn't replace it with anything.  You might want to add TestLoggerFactory.clear() or TestLogger.clear().  See the project's [usage docs](https://projects.lidalia.org.uk/slf4j-test/usage.html).

Of course the real fix would be for someone in that project to create a JUnit5 Extension to replace the JUnit4 Rule.  That doesn't look like it would be too hard however the project looks like it might be inactive.  No code commits in the last 4 years.